cmd/antrea-agent/agent.go

```diff
@@ -253,6 +255,8 @@ func run(o *Options) error {
 
 	log.StartLogFileNumberMonitor(stopCh)
 
+	go routeClient.Run(stopCh, ipTablesSyncInterval)
```

Comment:
Agreed @tnqn , this should also stay internal. I missed this in previous iteration. I was thinking of exposing a function `SetSyncInterval` which would override the variable, it would be meant to be called from route package's Initialization() method and from the integration test.
Let me know if this is inconsistent. Also, if there's someplace in antrea code where we have a similar issue, I can look at that and change this part.

Reply:
I forgot that the integration test is in a different package. I'm ok with just declaring the variable as public, but we should comment why it's made so. We had a similar case before that an integration test needs to use package internal consistant/variable and it just made them public to solve it. However, that code has been deleted when implmenetation evolves.

@antoninbas what do you think?

Reply:
typically my favorite approach is to have a way to pass it to the constructor:

```go
const ipTablesSyncInterval = 60 * time.Second

type ClientConfig struct {
    IPTablesSyncInterval time.Duration
}

func NewClient(serviceCIDR *net.IPNet, networkConfig *config.NetworkConfig, noSNAT bool, configFns ...func(*ReporterConfig)) *Client {
    config := ClientConfig{
        IPTablesSyncInterval: defaultIPTablesSyncInterval,
    }
    for _, fn := range configFns {
        fn(&config)
    }
    return &Client{
        ClientConfig: config,
        // other members
    }
}
```
This way the call to `NewClient` does not need to change in `cmd/antrea-agent/agent.go`, but for the integration tests (or unit tests for that matter), we can easily provide a different value for the sync interval.

Of course that may be a bit overkill here with a single configuration parameter. So if you prefer to make the variable public and add a comment, that's fine by me.

Reply:
@antoninbas I actually missed this last comment, I just noticed it after push.